rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/participant.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RMW_FASTRTPS_SHARED_CPP__PARTICIPANT_HPP_
+#define RMW_FASTRTPS_SHARED_CPP__PARTICIPANT_HPP_
+
+#include "rmw/types.h"
+
+#include "rmw_dds_common/context.hpp"
+
+#include "rmw_fastrtps_shared_cpp/custom_participant_info.hpp"
+
+namespace rmw_fastrtps_shared_cpp
+{
+
+CustomParticipantInfo *
+create_participant(
+  const char * identifier,
+  size_t domain_id,
+  const rmw_security_options_t * security_options,
+  bool localhost_only,
+  const char * context_name,
+  const char * context_namespace,
+  rmw_dds_common::Context * common_context);
+
+rmw_ret_t
+destroy_participant(CustomParticipantInfo * info);
+}  // namespace rmw_fastrtps_shared_cpp
```

Comment:
@ivanpauno nit: add space between function and closing namespace bracket.

Reply:
done